clouds/snowflake/modules/sql/h3/H3_CENTER.sql

```diff
@@ -1,31 +1,11 @@
 ----------------------------
--- Copyright (C) 2023 CARTO
+-- Copyright (C) 2021-2024 CARTO
 ----------------------------
```

Comment:
This is something small but we try to keep the `-` to the end. You can just copy the header from another file and modify it. This would apply to the rest of the files of this PR.

```
----------------------------
-- Copyright (C) 2021-2024 CARTO
----------------------------
```
->
```
--------------------------------
-- Copyright (C) 2021-2024 CARTO
--------------------------------
```